internal/controller/grafanaorganization_controller.go

```diff
@@ -165,20 +175,26 @@ func (r GrafanaOrganizationReconciler) reconcileCreate(ctx context.Context, graf
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
-	var lastError error
+	var errs []error
 
 	// Update the datasources in the CR's status
 	if err := r.configureDatasources(ctx, grafanaOrganization); err != nil {
-		lastError = err
+		errs = append(errs, err)
 	}
 
 	// Configure Grafana RBAC
 	if err := r.configureGrafanaSSO(ctx); err != nil {
-		lastError = err
+		errs = append(errs, err)
+	}
+
+	// Configure up the alloy-rules app
+	if err := r.AlloyRulesService.Configure(ctx); err != nil {
+		logger.Error(err, "failed to configure alloy-rules")
```

Comment:
I would rather not have this extra log line since the error is already returned and logged later on.

Reply:
```suggestion
```